kimchi/js/bindings.js

```diff
@@ -2259,12 +2265,60 @@ function caml_pasta_fq_poseidon_block_cipher(_fake_params, fq_vector) {
     });
 }
 
-// Provides: caml_pasta_fp_plonk_proof_example_with_lookup
-function caml_pasta_fp_plonk_proof_example_with_lookup() {
-    // This is only used in the pickles unit tests
-    throw new Error("Unimplemented caml_pasta_fp_plonk_proof_example_with_lookup");
-}
 
 // Provides: prover_to_json
 // Requires: plonk_wasm
 var prover_to_json = plonk_wasm.prover_to_json;
+
+// Provides: integers_uint64_of_uint32
+function integers_uint64_of_uint32(i) {
+    // Same as integers_uint64_of_int
+    return new new UInt64(caml_int64_of_int32(i));
```

Comment:
there are two `new`s here, and `UInt64` as well as `caml_int64_of_int32` seem to be missing as a dependency? You annotate a dependency with `// Requires: ...`

Reply:
Done

Reply:
This should be fixed.